Formula/libvisual-plugins-0.4.rb

```diff
@@ -0,0 +1,40 @@
+class LibvisualPlugins04 < Formula
```

Comment:
```suggestion
class LibvisualPlugins < Formula
```
Same comment as `libvisual`.